### PR TITLE
Request config override

### PR DESCRIPTION
**Pull request by medjedovicm**

### CODE REVIEW

File: src/SASjs.ts [hunk 1]
```diff
@@ -587,10 +596,9 @@ export default class SASjs {
     return sasjsWaitingRequest.requestPromise.promise;
   }
 
-  private async executeJobViaJes(
+  private async executeJobViaWeb(
```
Comment (allanbowe) [inline src/SASjs.ts:599]: isn't the custom config needed here also?

File: src/SASjs.ts [hunk 2]
```diff
@@ -383,40 +385,51 @@ export default class SASjs {
...
-      return await this.executeJobViaJes(
+      requestResponse = await this.executeJobViaWeb(
         sasJob,
         data,
-        params,
         loginRequiredCallback
       );
     }
+
+    this.sasjsConfig = this.globalSasjsConfig;
```
Comment (allanbowe) [inline src/SASjs.ts:430]: Why is this part necessary?

File: src/SASjs.ts [hunk 3]
```diff
@@ -42,7 +42,8 @@ const requestRetryLimit = 5;
  *
  */
 export default class SASjs {
-  private sasjsConfig = new SASjsConfig();
+  private globalSasjsConfig: SASjsConfig = new SASjsConfig();
+  private sasjsConfig: SASjsConfig = new SASjsConfig();
```
Comment (allanbowe) [inline src/SASjs.ts:46]: is this an empty config?  What would happen if the`globalSasConfig` is updated, but not the `sasjsConfig` ?

File: src/SASjs.ts [hunk 4]
```diff
@@ -814,8 +819,7 @@ export default class SASjs {
     for (const sasjsWaitingRequest of this.sasjsWaitingRequests) {
```
Comment (krishna-acondy) [inline src/SASjs.ts:819]: Would we not need to add the `config` to `SASjsWaitingRequest` as well?
